Makefile

```diff
@@ -182,6 +183,16 @@ integration: ## run integration tests
 	@echo "$(WHALE) $@"
 	@go test ${TESTFLAGS} -test.root -parallel ${TESTFLAGS_PARALLEL}
 
+# TODO integrate cri integration bucket with coverage
+bin/cri-integration.test: $(CRI_INTEGRATION_SOURCES)
```

Comment:
I'd suggest making this `.PHONY`. It is hard to track all dependency source files.